src/tests/Microsoft.VisualStudio.Composition.Tests/EmbeddedableTypesMixedTests.cs

```diff
@@ -20,7 +20,8 @@ namespace Microsoft.VisualStudio.Composition.Tests
     /// </summary>
     public class EmbeddedableTypesMixedTests
     {
-        [MefFact(CompositionEngines.V1Compat, "Microsoft.VisualStudio.Composition.NonEmbeddingTypeReceiver", typeof(EmbeddableTypesTests.PartThatExportsIVsProjectReference))]
+        [MefFact(CompositionEngines.V1Compat, "Microsoft.VisualStudio.Composition.NonEmbeddingTypeReceiver", typeof(EmbeddableTypesTests.PartThatExportsIVsProjectReference), SkipOnMono = true)]
+        [Trait("NoPIA", "SkipOnMono")]
```

Comment:
I can certainly change traits to look like that. However I haven't found a way to drive the skipping logic off of a trait.

Shall I still change the traits like you suggest above? I'm assuming I'll also be adding:

```
[Trait("AppDomains", "true")]
and
[Trait("WeakReference", "true")]
```
as other causes of test failures I've identified so far.